### ISSUE TITLE
Cleanup Entries "Run Unicode Converter..." Should Include Editor

### ISSUE THREAD

**Issue by jkulesza** [JabRef/jabref#1256]
JabRef version 3.2 on Mac OS X 10.11.4.

Steps to reproduce:

1. Create new book entry
2. Type "tëßt" into Title, Publisher, Year, Author, and Editor
3. Check BibTeX source:

```
@Book{,
  title =     {tësßt},
  publisher = {tësßt},
  year =      {tësßt},
  author =    {tësßt},
  editor =    {tësßt}
}
```

4. Highlight entry, Cleanup, ensure "Run Unicode converter on..." is checked, click OK.
5. Check BibTeX source:

```
@Book{,
  title =     {t{\"{e}}s{\ss}t},
  publisher = {tësßt},
  year =      {tësßt},
  author =    {t{\"{e}}s{\ss}t},
  editor =    {tësßt}
}
```

Editor could be a person's name requiring "cleanup."   Perhaps Unicode cleanup should be extended to other entries as well (e.g., Publisher).

**Comment by matthiasgeiger** [comment]
If you [update to the newly released version 3.3](http://www.jabref.org/blog/) you can customize the cleanup to match your needs:
![field-formatter](https://cloud.githubusercontent.com/assets/676652/14675024/332c5474-0707-11e6-84df-49c2507d6b4f.png)

**Comment by matthiasgeiger** [comment]
... and you can even tell JabRef to perform this automatically each time your database is saved using the save actions (available through: File -> Database properties).

**Comment by grimes2** [comment]
Well, I don't know the exact specification for BibTeX, but I did get the Parser error (in Lyx):
> BibTeX Parser: Unable to read value for field: publisher, for key: Paumgartner1957.

The reason is an Umlaut in publisher field. Same for editor and note field.

Maybe it is a good idea, to add
> editor: Unicode to Latex
publisher: Unicode to Latex
note: Unicode to Latex

as standard to "Run field formatter"

**Comment by jkulesza** [comment]
@matthiasgeiger Excellent, thanks.  Sorry for being 3 days behind the times.  

Is there a way to subscribe to a list to see when new JabRef releases are made (but otherwise receive no emails)?  Development is quite active (a great thing) but it is hard to keep track of what's going on without committing lots of time to studying the progress and changes.

In other Unicode issues, are the author names mangled in the About dialog or are they all as intended?

<img width="748" alt="screen shot 2016-04-20 at 9 26 30 am" src="https://cloud.githubusercontent.com/assets/9939059/14676127/11104de6-06da-11e6-87a9-4284a35acc50.png">

**Comment by Siedlerchr** [comment]
@jkulesza  Regarding  the authors, that is a bug in the gradle built system (external tool). The error will be fixed in the future,  #1178.

Maybe we should link to that issue on the homepage.
For Release: You could subscribe to the atom-RSS-Feed:
https://github.com/JabRef/jabref//releases.atom

**Comment by jkulesza** [comment]
@Siedlerchr Gotcha, thanks.  I'll close this but open a new ticket regarding a few things I've identified with 3.3.
